fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/LinkedOrganizationsRepository.kt

```diff
@@ -35,6 +34,36 @@ class LinkedOrganizationsRepository @Inject constructor() {
     return challenge
   }
 
+  fun addLinkedLao(laoId: String, otherLaoId: String, tokens: Array<String>) {
+    val laoMap = linkedLaos.getOrPut(laoId) { mutableMapOf() }
+    laoMap[otherLaoId] = tokens
+    onLinkedLaosUpdatedCallback?.invoke(laoId, laoMap)
+  }
+
+  fun updateAndNotifyLinkedLao(
+      laoId: String,
+      otherLaoId: String,
+      tokens: Array<String>,
+      rollCallId: String
+  ) {
+    addLinkedLao(laoId, otherLaoId, tokens)
+    newTokensNotifyFunction?.invoke(laoId, otherLaoId, rollCallId, tokens)
```

Comment:
If you take a look at other repos (any one, e.g. rollcallrepo) you'd see that in general this is not the preferred way to notify modifications on a given data structure, but that we use java rx constructs (subject, observables and so on)

Reply:
Well the thing is that I tried using observables and for some reason I had many problems with the UI not updating or weird other issues... That's why in the end I changed to using callbacks, I know it's a bit more complex but at least it's working

Reply:
Sure, we can keep it as is then